### DISCUSSION TITLE
Using Home / Away mode in homebridge vs Ring app

### DISCUSSION THREAD

**Discussion by slawnz** [dgreif/ring#1009] [General]
I’m seeing some odd behaviour when using the Home / Away mode in homebridge with my Ring cameras. After triggering a mode (etc Away) via the Home app, if I then go into the Ring app, the “modes” options normally available at the top of the main dashboard screen have disappeared, and the app acts as though the feature has not even been enabled before. If I select to enable them, the motion / live view options for each camera per Mode have been reset to the defaults. Is this expected behaviour?

**Comment by dgreif** [comment]
Hmm this doesn't sound like the expected behavior.  I have a Ring Alarm which behaves a little different from Modes, but Modes shouldn't be getting reset in the way you described.  Would you mind opening an issue and filling out the issue template with all these details?
